build.sh

```diff
@@ -1057,21 +1041,17 @@ case "$COMMAND" in
     # CocoaPods
     ######################################
     "cocoapods-setup")
-        if [[ "$2" != "swift" ]]; then
-          if [ ! -f core/version.txt ]; then
-            sh build.sh download-core
-          fi
+        if [ ! -f core/version.txt ]; then
```

Comment:
We don't want to do any of this for the RealmSwift pod, but now that there's *no* setup for it that should be done by removing the prepare_command from RealmSwift.podspec.

Reply:
If I read this right: resolved by 1195e5a.